File: CS/CheckBoxLegendExample/CheckBoxLegend/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace CheckBoxLegend
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            ChartLegendHelper helper = new CheckBoxLegend.ChartLegendHelper();
            helper.Attach(dashboardViewer1, "chartDashboardItem1");
            dashboardViewer1.LoadDashboard("nwindDashboard.xml");
        }
    }
}

File: VB/CheckBoxLegendExample/CheckBoxLegend/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace CheckBoxLegend
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
			Dim helper As ChartLegendHelper = New CheckBoxLegend.ChartLegendHelper()
			helper.Attach(dashboardViewer1, "chartDashboardItem1")
			dashboardViewer1.LoadDashboard("nwindDashboard.xml")
		End Sub
	End Class
End Namespace
